### DISCUSSION TITLE
Add Custom Module to Existing Orchard Core Project

### DISCUSSION THREAD

**Discussion by JStafford00** [OrchardCMS/OrchardCore#10438] [Q&A] [answered]
So I have been trying to add a custom module to an existing Orchard Core project in Visual Studio. I added a reference to the module and the module is set up to use Orchard Core (NuGet and Startup is correct). But it seems to not be able to access the module on the website. Am I missing something here?

**Comment by hishamco** [comment, marked as answer]
You need to go to Features menu to enable your referenced module

**Reply by JStafford00** [reply]
Everything that I saw wanted me to go to the Module menu under Design, but there wasn't a Module menu so assumed that was a older version. Yup it works now. Thank you!. Should have asked sooner, would have saved me 3 hours of searching for an answer and testing different ways.

**Reply by hishamco** [reply]
It happens ;)
